Notebook.ipynb

```diff
@@ -295,15 +318,15 @@
    "outputs": [],
    "source": [
     "while(not level.is_episode_over()):\n",
-    "    steinbeck.act(level, show_steps=True, graphic=True, delay=0.1)\n"
+    "    steinbeck.act(level, show_steps=True, graphic=True, delay=0.1)"
```

Comment:
yep this is a long abb movie ... and the messages are quite unreadable...

Reply:
It is what it is.
Life is life. 
Stay with me